### PR TITLE
Fix spelling

### PR DESCRIPTION
**Pull request by OrangeTux**
Thanks for this great project.

While digging through the code I found a few spelling errors. Here is a PR that fixes a few of them.
I hope you don't mind.

### CODE REVIEW

File: pyftpdlib/handlers.py [hunk 1]
```diff
@@ -1837,23 +1837,23 @@ def log(self, msg, logfun=logger.info):
...
+        """Log an error including additional identifying session data."""
         prefix = self.log_prefix % self.__dict__
         logger.error("%s %s" % (prefix, msg))
 
     def log_exception(self, instance):
         """Log an unhandled exception. 'instance' is the instance
         where the exception was generated.
         """
-        logger.exception("unhandled exception in instance %r", instance)
+        logger.excepftptftpion("unhandled exception in instance %r", instance)
```
Comment (lurch) [inline pyftpdlib/handlers.py:1856]: Ummmm??? :confused:

File: pyftpdlib/log.py [hunk 2]
```diff
@@ -81,8 +81,7 @@ def __init__(self, *args, **kwargs):
                 logging.WARNING: unicode(curses.tparm(fg_color, 3), "ascii"),
                 # red
                 logging.ERROR: unicode(curses.tparm(fg_color, 1), "ascii")
-            }
-            self._normal = unicode(curses.tigetstr("sgr0"), "ascii")
+            } self._normal = unicode(curses.tigetstr("sgr0"), "ascii")
```
Comment (lurch) [inline pyftpdlib/log.py:84]: Was this intentional?

File: pyftpdlib/handlers.py [hunk 3]
```diff
@@ -1773,7 +1773,7 @@ def push_dtp_data(self, data, isproducer=False, file=None, cmd=None):
         """
         if self.data_channel is not None:
             self.respond(
-                "125 Data connection already open. Transfer starting.")
+                "150 Data connection already open. Transfer starting.")
```
Comment (lurch) [inline pyftpdlib/handlers.py:1776]: https://en.wikipedia.org/wiki/List_of_FTP_server_return_codes suggests that `125` was the correct value here?

**Comment by lurch** [comment]
I'm not sure if `unicode` and `utf8` really needed "correcting", but the majority of the spelling corrections LGTM.

**Comment by OrangeTux** [comment]
I understand. I will revert those changes.

File: pyftpdlib/handlers.py (hunk 3 above)
Comment (OrangeTux) [reply pyftpdlib/handlers.py:1776]: You're totally right. This is a leftover of me playing around with the library and a buggy FTP client. I'll revert this.

File: pyftpdlib/log.py (hunk 2 above)
Comment (OrangeTux) [reply pyftpdlib/log.py:84]: Nope, will revert it.

File: pyftpdlib/handlers.py (hunk 1 above)
Comment (OrangeTux) [reply pyftpdlib/handlers.py]: Sorry ;) Will fix it.

**Comment by giampaolo** [comment]
Thank you.

**Comment by giampaolo** [comment]
Out of curiosity: did you use some tool to do this? If so, I would like to integrate it in my projects somehow.

**Comment by OrangeTux** [comment]
I use spell check function of neovim. You can read about it [here](https://www.linux.com/training-tutorials/using-spell-checking-vim/). It's nothing sophisticated. I just jumped through the list of misspellings using `]s`  and `[s`. Every misspelling I fixed manually.

**Comment by lurch** [comment]
@giampaolo I've used https://github.com/codespell-project/codespell on other projects.

File: pyftpdlib/handlers.py [hunk 4]
```diff
@@ -3005,7 +3005,7 @@ def ftp_OPTS(self, line):
 
     def ftp_NOOP(self, line):
         """Do nothing."""
-        self.respond("200 I successfully done nothin'.")
+        self.respond("200 I successfully did nothing'.")
```
Comment (lurch) [inline pyftpdlib/handlers.py:3008]: A little bit late, but I've just noticed that now that the abbreviated `nothin'` has been replaced with `nothing`, the apostrophe should be removed ;-)

**Comment by giampaolo** [comment]
Thanks Andrew.

**Pull request merged**